content/solutions/docker.adoc

```diff
@@ -19,7 +19,29 @@ ____
 
 What this offers to Jenkins users is a means to isolate different jobs from one another, quickly clean a job's workspace or even dynamically deploy or schedule jobs with Docker containers to increase resource utilization and efficiency.
 
+== Getting Started
+
+* link:https://docs.docker.com/get-started/[Getting Started with Docker]
+* link:https://hub.docker.com/r/jenkins/jenkins[Official Jenkins master image for Docker]
+* link:https://medium.com/@gustavo.guss/quick-tutorial-of-jenkins-b99d5f5889f2[Jenkins+Docker Tutorial]
+
+== Demos
+
+Demonstrations of Pipeline functionality you can run locally.
+
+* link:https://hub.docker.com/r/jenkinsci/workflow-demo/[continuous delivery]
+* link:https://hub.docker.com/r/jenkinsci/pipeline-as-code-github-demo/[GitHub organization folders]
+* link:https://hub.docker.com/r/jenkinsci/docker-workflow-demo/[build, test, and publish with Docker]
+
+== Presentations
+
+* link:https://www.youtube.com/watch?v=pMO26j2OUME[Run Jenkins in Docker Container]
+* link:https://www.youtube.com/watch?v=6tcoRIPBd8s[Jenkins - Build & Publish Docker Images]
+* link:https://www.youtube.com/watch?v=iN3he0eVUyw&t=31s[Jenkins on Docker: step by step for beginners]
+* link:https://www.youtube.com/watch?v=nMLQgXf8tZ0[DevOps project with Git, Jenkins and Docker on AWS]
+* link:https://www.youtube.com/watch?v=9TX5LOInPIQ[Deploy a war file on Docker container using Jenkins]
+* link:https://www.youtube.com/watch?v=z32yzy4TrKM[How to build Docker Image Using Jenkins Pipeline]
```

Comment:
The top of the page already provides a location and thumbnails for videos.  Wouldn't it be better to include this in the video thumbnails above rather than a list of links here.